x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -59,21 +60,41 @@ func (k Keeper) SettlePendingClaims(ctx sdk.Context) (
 
 		// NB: Not every (Req, Res) pair in the session is inserted into the tree due
 		// to the relay mining difficulty. This is the count of non-empty leaves that
-		// matched the necessary difficulty and is therefore an estimation of the total
-		// number of relays serviced and work done.
+		// matched the necessary difficulty.
 		numClaimRelays, err = claim.GetNumRelays()
 		if err != nil {
 			return settledResult, expiredResult, err
 		}
 
-		// DEV_NOTE: We are assuming that (numClaimComputeUnits := numComputeUnits * service.ComputeUnitsPerRelay)
+		// DEV_NOTE: We are assuming that (numClaimComputeUnits := numClaimRelays * service.ComputeUnitsPerRelay)
 		// because this code path is only reached if that has already been validated.
 		numClaimComputeUnits, err = claim.GetNumClaimedComputeUnits()
 		if err != nil {
 			return settledResult, expiredResult, err
 		}
 
-		// TODO(@red-0ne, #781): Convert numClaimedComputeUnits to numEstimatedComputeUnits to reflect reward/payment based on real usage.
+		// Get the relay mining difficulty for the service that this claim is for.
+		serviceId := claim.GetSessionHeader().GetServiceId()
+		relayMiningDifficulty, found := k.serviceKeeper.GetRelayMiningDifficulty(ctx, serviceId)
+		if !found {
+			relayMiningDifficulty = servicekeeper.NewDefaultRelayMiningDifficulty(ctx, logger, serviceId, 0)
+		}
+		// numEstimatedComputeUnits is the probabilistic estimation of the off-chain
+		// work done by the relay miner in this session. It is derived from the claimed
+		// work and the relay mining difficulty.
+		numEstimatedComputeUnits, err := claim.GetNumEstimatedComputeUnits(relayMiningDifficulty)
+		if err != nil {
+			return settledResult, expiredResult, err
+		}
+
+		sharedParams := k.sharedKeeper.GetParams(ctx)
+		// claimedAmount is the amount of uPOKT that the supplier would receive if the
+		// claim is settled. It is derived from the claimed number of relays, the service
+		// current minding difficulty and the global network parameters.
+		claimedAmount, err := claim.GetClaimeduPOKT(sharedParams, relayMiningDifficulty)
```

Comment:
NIT: Wdyt about `claimedCoin` or `claimeduPOKT` to avoid potential confusion regarding `sdk.Coin#Amount`?